plugins/gatsby-source-nav/gatsby-node.js

```diff
@@ -115,7 +113,6 @@ const createWhatsNewNav = async ({ createNodeId, nodeModel }) => {
     },
   });
 
-  const recentPosts = posts.slice(0, RECENT_POSTS_COUNT);
```

Comment:
We can probably get rid of the `concat` that uses this variable a few lines down too.

Reply:
good catch!